Assignment/one/CustomerOfShoppingCart.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Scanner;
+
+
+public class CustomerOfShoppingCart {
+	
+	public static void main(String ar[])
+	{
+		ShoppingCart cart=new ShoppingCart();
```

Comment:
variable name should be like **shoppingCartInstance or shoppintCartRecord** and these variable name should be in lower camel case